src/domain/api/client.ts

```diff
@@ -15,8 +16,9 @@ export default new ApolloClient({
     } catch (e) {
       // User not authenticated
       // eslint-disable-next-line no-console
+      console.error('Error in client.ts');
       console.error(e);
-      // TODO: add error-handler
+      toast('An error occured. Please try again later');
```

Comment:
We can use `i18n` instance to translate this, locale-based is not necessary but translation should be tracked.